app/controllers/idv/usps_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Idv
   class UspsController < ApplicationController
+    class TimeoutError < StandardError; end
```

Comment:
I'm really worried we'll get this confused with `::Timeout::Error`

also it seems redundant that we define it in a few places.

Can we add something like `LambaJobs::Runner::ProofingTimeout` and use that in all places instead?